cpp/scripts/run-clang-tidy.py

```diff
@@ -239,7 +265,7 @@ def main():
     if not os.path.exists(".git"):
         raise Exception("This needs to always be run from the root of repo")
     # Check whether clang-tidy exists
-    if shutil.which("clang-tidy") is not None:
+    if shutil.which("clang-tidy") is None:
         print("clang-tidy not found. Exiting...")
```

Comment:
Should we maybe raise an exception here instead? It seems to me that a silent pass could be problematic.

Reply:
Sounds fine to me.

Reply:
https://github.com/rapidsai/cuml/pull/5235/commits/403ff5c728d81181dcfe8904f2b3ef8a575302e1